packages/runtime/test/integration/pool-transaction-fee-test.ts

```diff
@@ -13,18 +13,20 @@ describe("Pooled Transaction Fees Test", function () {
 	let john = new AccountStore(1e30);
 	let alice = new AccountStore(minBalance);
 	let bob = new AccountStore(minBalance);
+	let elon = new AccountStore(minBalance);
 
 	let runtime: Runtime;
 
 	this.beforeEach(async function () {
-		runtime = new Runtime([john, alice, bob]); // setup test
+		runtime = new Runtime([john, alice, bob, elon]); // setup test
```

Comment:
lets use `runtime.defaultAccounts` instead

Reply:
We also need runtime later in the unit tests.